includes/Google/u2f-api.js

```diff
@@ -17,6 +17,12 @@
 var u2f = u2f || {};
 
 /**
+ * Check if browser supports U2F API before this wrapper was added.
+ * @type {int}
+ */
+u2f.HasNativeApiSupport = ( ( u2f && u2f.register ) || ( chrome && chrome.runtime ) );
```

Comment:
`u2f.register` should exist in Firefox with the extension enabled before we add our own `u2f.register` wrapper further down this file.